package.json

```diff
@@ -67,31 +79,40 @@
 			"globalThis"
 		],
 		"rules": {
-			"import/no-useless-path-segments": "off",
-			"unicorn/import-index": "off",
-			"import/extensions": "off"
+			"@typescript-eslint/dot-notation": "off",
+			"@typescript-eslint/quotes": "off",
+			"ava/no-import-test-files": "off",
+			"unicorn/filename-case": "off",
+			"unicorn/prevent-abbreviations": "off"
```

Comment:
Generally don't disable this. If it has false-positives, add inline `// eslint-disable..` comments.